workflow_step_execute.go

```diff
@@ -0,0 +1,74 @@
+package slack
+
+import (
+	"context"
+	"encoding/json"
+)
+
+type (
+	WorkflowStepCompletedRequest struct {
+		WorkflowStepExecuteID string             `json:"workflow_step_execute_id"`
+		Outputs               *map[string]string `json:"outputs"`
+	}
+
+	WorkflowStepFailedRequest struct {
+		WorkflowStepExecuteID string `json:"workflow_step_execute_id"`
+		Error                 struct {
+			Message string `json:"message"`
+		} `json:"error"`
+	}
+)
+
+// WorkflowStepCompleted indicates step is completed
+func (api *Client) WorkflowStepCompleted(workflowStepExecuteID string, outputs *map[string]string) error {
```

Comment:
[question]
What is the reason of using a pointer of map?

Reply:
How about receiving optional arguments like the following way?
https://commandcenter.blogspot.com/2014/01/self-referential-functions-and-design.html

Reply:
> I used pointer to represent empty because outputs is optional

FYI: Maybe, you can pass nil without pointer.

> Does this mean creating a function to set outputs and using to argument of WorkflowStepCompleted function?

Yes, that's right.

Reply:
Thank you for reviewing.
I pushed new commit.